bin/dummy_cuckoo_api.py

```diff
@@ -0,0 +1,133 @@
+        self.jobs = []
+
+    def register_job(self):
+        """ Adds a new job to the Queue and returns its id """
+        self.nextid = self.nextid + 1
+        self.jobs.append(self.nextid - 1)
+        print("New job created %d. Queue %s" % (self.nextid-1, self.jobs))
+        return self.nextid - 1
+
+    def delete_job(self, job_id):
+        """ Removes a job from the Queue """
+        self.jobs.remove(job_id)
+        print("Remove job {}. Queue %s".format(job_id) % self.jobs)
+
+
+class QueueResource(Resource):
+    """ Intermediate class to extend normal Resource by Queue """
+    def __init__(self, queue):
+        self.queue = queue
+
+
+class Status(QueueResource):
+    """ Resource to get Dummy Cuckoo's status on tasks """
+    def get(self):
+        """ Returns number of tasks in the respective state """
+        result = {'tasks':
+                      {
+                          'reported': self.queue.nextid - 1 - len(self.queue.jobs),
+                          'running': len(self.queue.jobs),
+                          'total': self.queue.nextid - 1,
+                          'dummyAPIqueue': self.queue.jobs
+                      }
+                 }
+        return flask.jsonify(result)
+
+
+class View(QueueResource):
+    """ Resource to access state of task """
+    def get(self, job_id):
+        """ Returns the current status of a given job_id.
+        Jobs finish by asking its state with a probability of 1/3 """
+        # job exists
+        if int(job_id) < self.queue.nextid:
+            result = {'task': {'status': 'reported'}}
+            # job is running
+            if int(job_id) in self.queue.jobs:
+                print("Job_id {} running".format(job_id))
+                result = {'task': {'status': 'running'}}
+                # job is done
+                if random.randint(0, 2) == 0:
+                    print("Job_id {} reported".format(job_id))
+                    result['task']['status'] = "reported"
+                    self.queue.delete_job(int(job_id))
+        else:
+            print("Job_id {} not found".format(job_id))
+            abort(404, message="Job_id {} doesn't exist".format(job_id))
+        return flask.jsonify(result)
+
+
+class Report(QueueResource):
+    """ Resource to accessed finished reports """
+    def get(self, job_id):
+        """ Unless Cuckoo report placed in storage/analyses/0/reports/report.json
+        returns minimal report """
+        try:
+            with open('storage/analyses/0/reports/report.json', 'r') as content_file:
+                report = json.load(content_file)
+        except FileNotFoundError:
+            report = {
+                "info": {
+                    "score": 10,
+                    "id": job_id,
+                },
+                "signatures": [
+                    {
+                        "description":
+                            "Malicious document featuring Office DDE has been identified",
```

Comment:
Brackets and string constant concatenation can be used here to break this long line.
``` python
>>> ("aa "
... "bb")
'aa bb'
```
Note the missing comma within the round brackets:
```
                "signatures": [
                    {
                        "description": (
                            "Malicious document featuring Office "
                            "DDE has been identified"
                        ),
                    }
                ],
```

Reply:
Addressed in new commits. Didn't know that you don't even need the brackets in a dict. Net.